CWE/decisions/claim_greater.txt

```diff
@@ -5483,7 +5475,7 @@ NOT = { government = democracy government = hms_government }
 			badboy = 12
 			set_country_flag = claim_greater_oceania
 			any_country = {
-				any_owned = { limit = { continent = oceania NOT = { is_core = XPI } } add_core = XPI } 
+				any_owned = { limit = { NOT = { is_core = XPI } OR = { continent = oceania } } add_core = XPI } 
```

Comment:
all_core = { limit = { NOT = { is_core = XPI } continent = oceania } add_core = XPI }